app/views/CreateChannelView/index.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useCallback, useEffect, useLayoutEffect } from 'react';
+import { shallowEqual, useDispatch } from 'react-redux';
+import { FlatList, ScrollView, StyleSheet, Text, View } from 'react-native';
+import { RouteProp, useNavigation, useRoute } from '@react-navigation/native';
+import { StackNavigationProp } from '@react-navigation/stack';
+import { useForm } from 'react-hook-form';
+
+import { useAppSelector } from '../../lib/hooks';
+import { sendLoadingEvent } from '../../containers/Loading';
+import { createChannelRequest } from '../../actions/createChannel';
+import { removeUser as removeUserAction } from '../../actions/selectedUsers';
+import KeyboardView from '../../containers/KeyboardView';
+import scrollPersistTaps from '../../lib/methods/helpers/scrollPersistTaps';
+import I18n from '../../i18n';
+import StatusBar from '../../containers/StatusBar';
+import { useTheme } from '../../theme';
+import { Review } from '../../lib/methods/helpers/review';
+import SafeAreaView from '../../containers/SafeAreaView';
+import sharedStyles from '../Styles';
+import { ChatsStackParamList } from '../../stacks/types';
+import Button from '../../containers/Button';
+import { ControlledFormTextInput } from '../../containers/TextInput';
+import { IOtherUser, UserItem } from './UserItem';
+import { RoomSettings } from './RoomSettings';
+
+const styles = StyleSheet.create({
+	container: {
+		flex: 1
+	},
+	list: {
+		width: '100%'
+	},
+	invitedHeader: {
+		marginTop: 18,
```

Comment:
Not a multiple of 4 and not the same spacing as seen on Figma.
The total spacing you're applying is 26 and should be 24